source/speechXml.py

```diff
@@ -47,7 +48,10 @@ def _buildInvalidXmlRegexp():
 			trailing=trailingSurrogate))
 
 RE_INVALID_XML_CHARS = _buildInvalidXmlRegexp()
-REPLACEMENT_CHAR = textUtils.REPLACEMENT_CHAR
+# remove this in 2022.1, instead use textUtils.REPLACEMENT_CHAR
+if version_year < 2022:
+	REPLACEMENT_CHAR = textUtils.REPLACEMENT_CHAR
+

```

Comment:
An optional change, that needs documenting

Reply:
I think I'll leave this. Given the context of how this is used I think it is ok for it to be part of the public API for speechXML